vivid-client/src/components/styled/Message.tsx

```diff
@@ -1,13 +1,26 @@
 import React from 'react';
 import './Message.css';
+import { Button } from './Button';
+import { useParams } from 'react-router-dom';
+import { useMessages } from '../../hooks/useMessages';
+import { useFetch } from '../../hooks/useFetch';
 
 export function Message(props: {
+  messageId: string;
   username: string;
   tag?: string;
   userColors: string[];
   messages: string[];
   blocked: boolean;
+  owner: boolean;
 }) {
+  const { id }: any = useParams();
```

Comment:
pass channelId through a prop instead of params. (when we use this component for friend DM's, the channel id wont be in the params)